index.bs

```diff
@@ -560,79 +560,246 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
+
+		To <dfn>create a {{ClipboardItem}} object</dfn>, given a [=clipboard item=] |clipboardItem| and a [=global object/Realm=] |realm|, run these steps:
+			1. Let |clipboardItemObject| be a [=new=] {{ClipboardItem}} with |realm|.
+
+			1. Set |clipboardItemObject|'s [=clipboard item=] to |clipboardItem|.
+
+		The <dfn constructor for="ClipboardItem" lt="ClipboardItem(items, options)"><code>new ClipboardItem(<var>items</var>, <var>options</var>)</code></dfn> constructor steps are:
+			1. Set [=this=]'s [=ClipboardItem/clipboardItem=] to a new [=clipboard item=].
+
+			1. Set [=this=]'s [=ClipboardItem/clipboardItem=]'s [=presentation style=] to |options|["{{ClipboardItemOptions/presentationStyle}}"].
+
+			1. Let |types| be a list of {{DOMString}}.
+
+			1. For each (|key|, |value|) in |items|:
+
+				1. Let |mimeType| be the result of [=parsing a MIME type=] given |key|.
+
+				1. If |mimeType| is failure, then throw a {{TypeError}}.
+
+				1. Let |representation| be a new [=representation=].
+
+				1. Set |representation|'s [=representation/MIME type=] to |key|.
+
+				1. Set |representation|'s [=representation/data=] to |value|.
+
+				1. Append |representation| to [=this=]'s [=ClipboardItem/clipboardItem=]'s [=list of representations=].
+
+				1. Let |mimeTypeString| be the result of [=serializing a MIME type=] with |mimeType|.
+				
+				1. Add |mimeTypeString| to |types|.
+
+			1. Set [=this=]'s [=ClipboardItem/types array=] to the result of running [=create a frozen array=] from |types|.
+
+		<h4 attribute for=ClipboardItem lt=presentationStyle>presentationStyle</h4>
+		<p>
+			The {{ClipboardItem/presentationStyle}} getter steps are to return [=this=]'s [=ClipboardItem/clipboardItem=]'s [=presentation style=].
+		</p>
+
+		<h4 attribute for=ClipboardItem lt=types>types</h4>
+		<p>
+			The {{ClipboardItem/types}} getter steps are to return [=this=]'s [=ClipboardItem/types array=].
+		</p>
+
+		<h4 method for=ClipboardItem lt=getType(type)>getType(type)</a> must run the below steps:</h4>
+
+			1. Let |realm| be [=this=]'s [=relevant realm=].
+
+			1. Let |mimeType| be the result of [=parsing a MIME type=] given |type|.
+
+			1. If |mimeType| is failure, then throw a {{TypeError}}.
+
+			1. Let |itemTypeList| be [=this=]'s [=ClipboardItem/clipboardItem=]'s [=list of representations=].
+
+			1. Let |p| be [=a new promise=] in |realm|.
+
+			1. For each |representation| in |itemTypeList|:
+
+				1. If |representation|'s [=representation/MIME type=] is |mimeType|, then:
+
+					1. Let |p1| be the |representation|'s [=representation/data=].
+
+						1. [=promise/React=] to |p1|:
+
+							1. If |p1| was fulfilled with value |v|, then:
+
+								1. If |v| is a {{DOMString}}, then follow the below steps:
+
+									1. Let |v1| be the result of [=/encoding=] |v|.
+
+									1. Let |blobData| be a [=Blob=] created using |v1| with its {{Blob/type}} set to |mimeType|, [=serialize a MIME type|serialized=].
```

Comment:
The link created by `[=Blob=]` points to https://w3c.github.io/FileAPI/#dfn-blob instead of https://w3c.github.io/FileAPI/#dfn-Blob. I guess the definition of the latter, in the File API should be changed to lowercase.

Reply:
Try using `{{Blob}}`.

Reply:
Done.